index.test.js

```diff
@@ -0,0 +1,85 @@
+const server = require("./api/server.js");
+const gamesList = require("./api/db.js");
+const request = require("supertest");
+
+describe("testing for GET and POST server endpoints", () => {
+  describe("GET /games endpoint tests", () => {
+    it("should return status code 200(OK)", async () => {
+      const response = await request(server).get("/games");
+      expect(response.status).toBe(200);
+    });
+
+    it("should return JSON", async () => {
+      const response = await request(server).get("/games");
+
+      expect(response.type).toBe("application/json");
+    });
+    it("should return the correct array of game objects", async () => {
+      const response = await request(server).get("/games");
```

Comment:
Since you make this same request each time for your in this `describe` tests, you should use a `beforeAll` instead so that you won't have to make multiple requests (and so that your tests will be quicker)! For example:
```javascript
describe("GET /games endpoint tests", () => {
  let response;
  beforeAll(async () => {
    response = await request(server).get("/games");
  });

  it("should return status code 200(OK)", () => {
    expect(response.status).toBe(200);
  });

  it("should return JSON", () => {
    expect(response.type).toBe("application/json");
  });

  // ...
```

Reply:
The same applies to your first three POST tests. Use a `beforeAll` for them!